### ISSUE TITLE
Rule proposal: `call-arguments-length`

### ISSUE THREAD

**Issue by fisker** [sindresorhus/eslint-plugin-unicorn#1356]
A generic rule to enforce correct call arguments length.

For example [CanvasRenderingContext2D.drawImage()](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/drawImage) can config as `{'*.drawImage': [3, 5, 9]}`, so it must be called with `3` or `5` or `9` arguments.

```js
// Fail
context.drawImage(image, dx);
context.drawImage(image, dx, dy, dWidth, );

// Pass
context.drawImage(image, dx, dy);
```

This rule accepts a configurable list, for each method of function, value can be a number or a range, or allowed lengths.

```js
{
	rules: {
		'unicorn/call-arguments-length':[
			'error',
			{
				'parseInt': 2,
				'*.drawImage': [3, 5, 9],
				'*.addEventListener': {min: 2}
			}
		]
	}
}

Ref: https://github.com/sindresorhus/eslint-plugin-unicorn/issues/974#issuecomment-768746222

**Comment by sindresorhus** [comment]
I think we should ship with a default preset for this rule that covers a lot of built-ins.

**Comment by sindresorhus** [comment]
This is now accepted.
